packages/expo-modules-autolinking/src/index.ts

```diff
@@ -83,6 +87,44 @@ function registerPatchReactImportsCommand() {
     .action(patchReactImportsAsync);
 }
 
+/**
+ * Registry the `react-native-config` command.
+ */
+function registerReactNativeConfigCommand() {
+  return commander
+    .command('react-native-config [paths...]')
+    .option(
+      '-p, --platform [platform]',
+      'The platform that the resulting modules must support. Available options: "android", "ios"',
```

Comment:
Other commands are using `apple` instead of `ios`. Maybe we should change it here too? cc @tsapeta

Reply:
in the mean time, i'm not sure how tvos and macos support out-of-tree platform autolinking. i guess they may have other cli / autolinking than rnc-cli?

Reply:
As far as I know, they don't. The community CLI supports out-of-tree platforms.
I like the idea of having `apple` platform instead, but we can change that later at some point.

Reply:
Cool, good to know. I think it's fine to use `ios` flag right now, but let's change it to `apple` later and make it feel out-of-tree platform support 😄 
Luckily, as opposed to apps, libraries can have only `android` and `apple` folders (i.e. there is no separate dir for macOS) and have a single pod that supports all the platforms. From autolinking perspective, it shouldn't matter what's the platform since all Apple platforms are handled in the same way.

For the config, let's stay with `ios` for compatibility reasons, but this is something that should be discussed with the RN team.